app/policies/group_policy.rb

```diff
@@ -48,6 +48,14 @@ def invite?
     user.is_admin? or user.owns(record) or member? && [Group::MEMBERSHIP[:open], Group::MEMBERSHIP[:invitation]].include?(record.membership)
   end
 
+  def add?
+    user.is_admin?
+  end
+
+  def remove?
+    user.is_admin?
```

Comment:
I wonder if we should also allow group owners to remove members (even if the group owner is not an admin)...

Reply:
I wouldn't mind that.